ARFrameMetadata/Metal/MetalCameraSession.swift

```diff
@@ -0,0 +1,294 @@
+//
+//  MetalCameraSession.swift
+//  MetalShaderCamera
+//
+//  Created by Jerónimo Valli on 9/18/20.
+//  Copyright © 2020 tokbox. All rights reserved.
+//
```

Comment:
We try to remove this headers from public code

Reply:
Removed!